Makefile

```diff
@@ -11,7 +11,7 @@ LDFLAGS += -X "github.com/pingcap/tidb/util/printer.TiDBGitHash=$(shell git rev-
 
 TARGET = ""
 
-.PHONY: godep deps all build install parser clean todo test tidbtest mysqltest gotest interpreter
```

Comment:
`. PHONY` section is only to support `make section` conflicting with `dir or filename`. It will not make server.